### ISSUE TITLE
Weird error at shutdown

### ISSUE THREAD

**Issue by lahwaacz** [ClearcodeHQ/pytest-postgresql#109]
I'm getting the following error at the end of pytest's output when a test fails. It doesn't prevent a successful pass when everything is OK, though.
```
********************************************************************************
Exception while deleting Executor. 'It is strongly suggested that you use
it as a context manager instead.
********************************************************************************
Exception ignored in: <bound method SimpleExecutor.__del__ of <pytest_postgresql.executor.PostgreSQLExecutor: "/usr/bin/p..." 0x7fe758d124a8>>
Traceback (most recent call last):
  File "/home/lahwaacz/Bbox/pg/python3/wiki-scripts/.tox/py36/lib/python3.6/site-packages/mirakuru/base.py", line 361, in __del__
  File "/home/lahwaacz/Bbox/pg/python3/wiki-scripts/.tox/py36/lib/python3.6/site-packages/mirakuru/base.py", line 314, in kill
  File "/home/lahwaacz/Bbox/pg/python3/wiki-scripts/.tox/py36/lib/python3.6/site-packages/mirakuru/base.py", line 227, in _kill_all_kids
  File "/home/lahwaacz/Bbox/pg/python3/wiki-scripts/.tox/py36/lib/python3.6/site-packages/mirakuru/base_env.py", line 52, in processes_with_env_psutil
  File "/home/lahwaacz/Bbox/pg/python3/wiki-scripts/.tox/py36/lib/python3.6/site-packages/psutil/__init__.py", line 1426, in process_iter
  File "/home/lahwaacz/Bbox/pg/python3/wiki-scripts/.tox/py36/lib/python3.6/site-packages/psutil/__init__.py", line 1371, in pids
  File "/home/lahwaacz/Bbox/pg/python3/wiki-scripts/.tox/py36/lib/python3.6/site-packages/psutil/_pslinux.py", line 1324, in pids
  File "/home/lahwaacz/Bbox/pg/python3/wiki-scripts/.tox/py36/lib/python3.6/site-packages/psutil/_pslinux.py", line 214, in get_procfs_path
KeyError: 'psutil'
```

**Comment by dlcron** [comment]
I'll attach minimal code example proving the case:

```
from pytest_postgresql import factories as postgresql_factories

postgres_proc = postgresql_factories.postgresql_proc(
    executable='/usr/lib/postgresql/9.5/bin/pg_ctl', port='5111',
)
postgresdb = postgresql_factories.postgresql('postgres_proc', db='db')


def test_abc(postgresdb):
    assert False
```

**Comment by lahwaacz** [comment]
https://github.com/fizyk/pytest-postgresql/commit/90b27a76ec10fc2bd0d713de488e9c5caed2f7d3 did __not__ fix the issue, there is still the same warning even with v1.3.3.

**Comment by fizyk** [comment]
okay, will look at it later this week probably

**Comment by fizyk** [comment]
I'm unable to reproduce this. Even with provided test. Could you post a pip freeze output?

**Comment by lahwaacz** [comment]
Oh, I was not reproducing it with the test by @dlcron above, but the test suite in [wiki-scripts](https://github.com/lahwaacz/wiki-scripts). It seems that your commit has fixed the above test, but if you replace `test_abc(postgresdb)` with `test_abc(postgres_proc)` you should get the error.

**Comment by fizyk** [comment]
okay, got the error.

**Comment by fizyk** [comment]
please try #123. That should fix it.

**Comment by lahwaacz** [comment]
Indeed, seems like that solves it. Thanks!

**Comment by fizyk** [comment]
released as 1.3.4 Will be uploaded to pypi in a few minutes
